tests/test_so101_camera_capture_cli.py

```
@@ -21,32 +22,113 @@ def test_parse_index_or_path_converts_plain_integer(self) -> None:
         self.assertEqual(report["outcome"], "NO_GO")
         self.assertFalse(report["pass_conditions"]["dependencies_available"])
         self.assertFalse(report["pass_conditions"]["frame_captured"])
         self.assertTrue(report["pass_conditions"]["trace_only_motion_boundary_preserved"])
         self.assertIn("lerobot", report["detail"])
+
+    def test_artifact_paths_must_be_repo_relative(self) -> None:
+        base = ROOT / "runs" / "physical"
+        base.mkdir(parents=True, exist_ok=True)
+        with TemporaryDirectory(dir=base) as tmpdir:
+            report_path = Path(tmpdir) / "absolute_report.json"
+            result = subprocess.run(
```

Ruff (0.15.18)

[error] `subprocess` call: check for execution of untrusted input

(S603)